### PR TITLE
media-libs/vidstab: new ebuild

### PR DESCRIPTION
**Pull request by thinrope**
This is a second attempt for PR, it was discussed in
https://github.com/gentoo/gentoo/pull/8250 before.

This is a standalone library used by ffmpeg, so submitting it first
here, once in the tree, I'll submit PR to add it (compile time) to
ffmpeg  (currently tested fine with ffmpeg-{3.3.6,4.1}).

If anyone can hint me in the right direction to enable the tests in
src_test, I'd do the rest. Basically there is one unit test (13 checks)
in tests/ and to manually run them, one needs to run from ${S}:

cd tests/cmake
cmake ../
make
./tests

But after fiddling with cmake-utils_src_test I couldn't figure it out...

Closes: https://bugs.gentoo.org/517892
Signed-off-by: Kalin KOZHUHAROV <kalin@thinrope.net>

### CODE REVIEW

File: media-libs/vidstab/vidstab-1.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2018-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (candrews) [inline media-libs/vidstab/vidstab-1.1.0.ebuild]: Please use EAPI=7

**Comment by candrews** [comment]
Can you also please make a -9999 ebuild that's an exact copy of the other one? You can use a structure like:
```
if [[ ${PV} == *9999 ]] ; then
        EGIT_REPO_URI="https://github.com/netdata/${PN}.git"
        inherit git-r3
else
        SRC_URI="https://github.com/netdata/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
        KEYWORDS="~amd64 ~x86"
fi
```
for example, allowing the two ebuild versions to have the same contents.

File: media-libs/vidstab/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```
Comment (candrews) [inline media-libs/vidstab/metadata.xml]: Please add:
```
    <upstream>
        <remote-id type="github">georgmartius/vid.stab</remote-id>
    </upstream>
```

**Comment by candrews** [comment]
Submitted a PR for them to return the correct exit code when tests pass: https://github.com/georgmartius/vid.stab/pull/69

File: media-libs/vidstab/metadata.xml (hunk 2 above)
Comment (thinrope) [reply media-libs/vidstab/metadata.xml]: done

**Comment by candrews** [comment]
Also, the commit message isn't valid, see https://www.gentoo.org/glep/glep-0066.html#commit-messages

File: media-libs/vidstab/vidstab-1.1.0.ebuild (hunk 1 above)
Comment (thinrope) [reply media-libs/vidstab/vidstab-1.1.0.ebuild]: done

**Comment by candrews** [comment]
@thinrope if you're okay with #10799 I'll merge it :)

**Comment by thinrope** [comment]
I am closing this PR, let's work on #10799 , test/fix ${S} for the 9999 version there and we should be done.

**Pull request closed without merging**